In [1]:
import pandas as pd
data = pd.read_csv('The Rise Of Artificial Intellegence2.csv')

In [2]:
data

Year  AI Software Revenue(in Billions)  \
0  2018                             10.10   
1  2019                             14.69   
2  2020                             22.59   
3  2021                             34.87   
4  2022                             51.27   
5  2023                             70.94   
6  2024                             94.41   
7  2025                            126.00   

   Global AI Market Value(in Billions) AI Adoption (%) Organizations Using AI  \
0                                 29.5             10%                    35%   
1                                 35.0             20%                    37%   
2                                 45.0             30%                    40%   
3                                 65.0             35%                    42%   
4                                140.0             35%                    45%   
5                                279.0             35%                    48%   
6                                400.0             40%                    50%   
7                               1810.0             63%                    55%   

  Organizations Planning to Implement AI  \
0                                    40%   
1                                    42%   
2                                    45%   
3                                    47%   
4                                    50%   
5                                    53%   
6                                    55%   
7                                    58%   

  Global Expectation for AI Adoption (%)  \
0                                    40%   
1                                    47%   
2                                    54%   
3                                    61%   
4                                    68%   
5                                    73%   
6                                    78%   
7                                    82%   

  Estimated Jobs Eliminated by AI (millions)  \
0                                        10%   
1                                        15%   
2                                        20%   
3                                        25%   
4                                        30%   
5                                        35%   
6                                        40%   
7                                        45%   

  Estimated New Jobs Created by AI (millions) Net Job Loss in the US  ...  \
0                                          5%                     7%  ...   
1                                          8%                  6.50%  ...   
2                                         12%                     6%  ...   
3                                         16%                  5.50%  ...   
4                                         20%                     5%  ...   
5                                         24%                  4.80%  ...   
6                                         28%                  4.50%  ...   
7                                         32%                  4.20%  ...   

  Estimated Revenue Increase from AI (trillions USD)  \
0                                                1.2   
1                                                1.8   
2                                                2.4   
3                                                3.0   
4                                                3.6   
5                                                4.2   
6                                                4.8   
7                                                5.5   

  Marketers Believing AI Improves Email Revenue  \
0                                        41.29%   
1                                           42%   
2                                           43%   
3                                           44%   
4                                           45%   
5                                           46%   
6                                           47%   
7                                

In [3]:
data.columns

Index(['Year', 'AI Software Revenue(in Billions)',
       'Global AI Market Value(in Billions)', 'AI Adoption (%)',
       'Organizations Using AI', 'Organizations Planning to Implement AI',
       'Global Expectation for AI Adoption (%)',
       'Estimated Jobs Eliminated by AI (millions)',
       'Estimated New Jobs Created by AI (millions)', 'Net Job Loss in the US',
       'Organizations Believing AI Provides Competitive Edge',
       'Companies Prioritizing AI in Strategy',
       'Estimated Revenue Increase from AI (trillions USD)',
       'Marketers Believing AI Improves Email Revenue',
       'Expected Increase in Employee Productivity Due to AI (%)',
       'Americans Using Voice Assistants (%)',
       'Digital Voice Assistants (billions of devices)',
       'Medical Professionals Using AI for Diagnosis',
       'AI Contribution to Healthcare(in Billions)',
       'Jobs at High Risk of Automation - Transportation & Storage (%)',
       'Jobs at High Risk of Automation - Whole

Bộ dữ liệu này cung cấp thông tin về xu hướng phát triển và tác động của trí tuệ nhân tạo (AI) tại USA từ năm 2018 đến 2025. Nó bao gồm các chỉ số như:


Doanh thu và giá trị thị trường AI: Sự tăng trưởng trong doanh thu phần mềm và giá trị thị trường AI toàn cầu.

Tỷ lệ chấp nhận AI: Mức độ triển khai AI trong các tổ chức và kỳ vọng toàn cầu về việc áp dụng AI.

Tác động đến việc làm: Số lượng việc làm bị loại bỏ và việc làm mới được tạo ra bởi AI.

Lợi thế cạnh tranh: Tỷ lệ các tổ chức tin rằng AI mang lại lợi thế cạnh tranh và việc ưu tiên AI trong chiến lược của các công ty.

Tăng trưởng doanh thu và năng suất: Sự gia tăng doanh thu và năng suất nhân viên nhờ AI.

Ứng dụng AI trong các lĩnh vực khác: Sử dụng AI trong chăm sóc sức khỏe, trợ lý giọng nói và các ngành có nguy cơ tự động hóa cao.

Bộ dữ liệu này cho thấy AI đang ngày càng trở thành một phần quan trọng trong các chiến lược kinh doanh và cuộc sống, với cả cơ hội và thách thức đi kèm.


AI Software Revenue(in Billions): Doanh thu phần mềm AI (tỷ USD)

Global AI Market Value(in Billions): Giá trị thị trường AI toàn cầu (tỷ USD)

AI Adoption (%): Tỷ lệ chấp nhận AI (%)

Organizations Using AI: Tổ chức sử dụng AI

Organizations Planning to Implement AI: Tổ chức dự định triển khai AI

Global Expectation for AI Adoption (%): Kỳ vọng toàn cầu về việc chấp nhận AI (%)

Estimated Jobs Eliminated by AI (millions): Ước tính việc làm bị AI loại bỏ (triệu)

Estimated New Jobs Created by AI (millions): Ước tính việc làm mới do AI tạo ra (triệu)

Net Job Loss in the US: Mất việc ròng tại Mỹ

Organizations Believing AI Provides Competitive Edge: Tổ chức tin rằng AI mang lại lợi thế cạnh tranh

Companies Prioritizing AI in Strategy: Công ty ưu tiên AI trong chiến lược

Estimated Revenue Increase from AI (trillions USD): Ước tính tăng doanh thu từ AI (nghìn tỷ USD)

Marketers Believing AI Improves Email Revenue: Nhà tiếp thị tin rằng AI cải thiện doanh thu email

Expected Increase in Employee Productivity Due to AI (%): Tăng năng suất nhân viên dự kiến nhờ AI (%)

Americans Using Voice Assistants (%): Người Mỹ sử dụng trợ lý giọng nói (%)

Digital Voice Assistants (billions of devices): Trợ lý giọng nói kỹ thuật số (tỷ thiết bị)

Medical Professionals Using AI for Diagnosis: Chuyên gia y tế sử dụng AI để chẩn đoán

AI Contribution to Healthcare(in Billions): Đóng góp của AI vào chăm sóc sức khỏe (tỷ USD)

Jobs at High Risk of Automation - Transportation & Storage (%): Việc làm có nguy cơ tự động hóa cao - Vận tải & Kho bãi (%)

Jobs at High Risk of Automation - Wholesale & Retail Trade: Việc làm có nguy cơ tự động hóa cao - Bán buôn & Bán lẻ

Jobs at High Risk of Automation - Manufacturing: Việc làm có nguy cơ tự động hóa cao - Sản xuất

In [4]:
import pandas as pd
import plotly.express as px
import plotly.graph_objects as go

In [5]:

# Biểu đồ cho AI Software Revenue và Global AI Market Value
fig1 = px.line(data, x='Year', y=['AI Software Revenue(in Billions)', 'Global AI Market Value(in Billions)'],
               labels={'value': 'Revenue/Market Value (Billions)', 'variable': 'Legend'},
               title='AI Software Revenue and Global AI Market Value Over Years')
fig1.show()

In [6]:
fig3d = go.Figure(data=[go.Surface(
    z=[data['AI Software Revenue(in Billions)'], data['Global AI Market Value(in Billions)']],
    x=data['Year'],
    y=['AI Software Revenue', 'Global AI Market Value']
)])
fig3d.update_layout(
    title='AI Software Revenue and Global AI Market Value (3D)',
    scene=dict(
        xaxis_title='Year',
        yaxis_title='Category',
        zaxis_title='Value (Billions USD)'
    )
)
fig3d.show()

Doanh Thu Phần Mềm AI: Tăng trưởng đều đặn, từ 10.1 tỷ USD năm 2018 lên khoảng 126 tỷ USD vào năm 2025

Giá Trị Thị Trường AI Toàn Cầu: Tăng mạnh, đạt 1.81 nghìn tỷ USD vào năm 2025.

In [7]:
# Biểu đồ cho AI Adoption và Organizations Using AI
fig2 = px.line(data, x='Year', y=['AI Adoption (%)', 'Organizations Using AI'],
              title='AI Adoption and Organizations Using AI Over Years')
fig2.show()

In [8]:
fig3d = go.Figure(data=[go.Surface(
    z=[data['AI Adoption (%)'], data['Organizations Using AI']],
    x=data['Year'],
    y=['AI Adoption', 'Organizations Using AI']
)])


fig3d.update_layout(
    title='AI Adoption and Organizations Using AI (3D)',
    scene=dict(
        xaxis_title='Year',
        yaxis_title='Category',
        zaxis_title='Percentage'
    )
)

fig3d.show()

Tỷ Lệ Chấp Nhận AI: Tăng từ 10% năm 2018 lên 63% vào năm 2025

Tổ Chức Sử Dụng AI: Tỷ lệ các tổ chức sử dụng AI tăng từ 35% năm 2018 lên 55% vào năm 2025.

In [9]:
# Biểu đồ thể hiện tương quan giữa công việc bị loại bỏ và công việc mới tạo bởi AI
fig = px.scatter(data,
                 x='Estimated Jobs Eliminated by AI (millions)',
                 y='Estimated New Jobs Created by AI (millions)',
                 text='Year',
                 labels={
                     'Estimated Jobs Eliminated by AI (millions)': 'Jobs Eliminated (Millions)',
                     'Estimated New Jobs Created by AI (millions)': 'New Jobs Created (Millions)'
                 },
                 title='Correlation Between Jobs Eliminated and New Jobs Created by AI')
fig.update_traces(textposition='top center')

In [10]:
fig3d = go.Figure(data=[go.Scatter3d(
    x=data['Year'],
    y=data['Estimated Jobs Eliminated by AI (millions)'],
    z=data['Estimated New Jobs Created by AI (millions)'],
    mode='markers+text',
    text=data['Year'],
    marker=dict(size=5, color=data['Year'], colorscale='Viridis', opacity=0.8),
)])

fig3d.update_layout(
    title='Correlation Between Jobs Eliminated and New Jobs Created by AI (3D)',
    scene=dict(
        xaxis_title='Year',
        yaxis_title='Jobs Eliminated (Millions)',
        zaxis_title='New Jobs Created (Millions)'
    )
)
fig3d.show()

Việc Làm và Tự Động Hóa:
Số việc làm bị loại bỏ do AI tăng, nhưng số việc làm mới do AI tạo ra cũng tăng, cho thấy sự chuyển đổi trong vai trò công việc.
Nguy cơ tự động hóa cao đối với các ngành sản xuất, vận tải và bán lẻ.

In [11]:
#Biểu đồ thể hiện sự tin tưởng của các tổ chức về lợi thế cạnh tranh mà AI có thể mang lại
fig = px.line(data, x='Year',
              y=['Organizations Believing AI Provides Competitive Edge'],
              labels={'value': 'Percentage', 'variable': 'Legend'},
              title='Competitive Edge Over Years')
fig.show()

Nhiều tổ chức tin rằng AI mang lại lợi thế cạnh tranh, và xu hướng này ngày càng tăng.

In [12]:


#Biểu đồ thể hiện dự đoán về sự tác động của AI đến năng suất lao động
fig1 = px.line(data, x='Year', y='Expected Increase in Employee Productivity Due to AI (%)',
               labels={'Expected Increase in Employee Productivity Due to AI (%)': 'Productivity Increase (%)'},
               title='Expected Increase in Employee Productivity Due to AI')

fig1.show()

#Biểu đồ thể hiện sự tác động của AI đến doanh thu
fig2 = px.line(data, x='Year', y='Estimated Revenue Increase from AI (trillions USD)',
               labels={'Estimated Revenue Increase from AI (trillions USD)': 'Revenue Increase (Trillions USD)'},
               title='Estimated Revenue Increase from AI')

fig2.show()

AI dự kiến sẽ làm tăng đáng kể năng suất lao động và từ đó đóng góp vào tăng trưởng doanh thu.